target/linux/ramips/dts/mt7620n.dtsi

```diff
@@ -248,6 +248,14 @@
 				ralink,function = "uartlite";
 			};
 		};
+
+                wled_pins: wled {
```

Comment:
cd target/linux/ramips/dts/
FILE_TO_DCHECK=$(FILE_TO_CHECK=$(grep "<&gpio3 0" * | cut -d':' -f1); grep mt7620n.dtsi $FILE_TO_CHECK | cut -d':' -f1 ); grep -A1 wled $FILE_TO_DCHECK

CF-WR800N.dts:                  ralink,group = "ephy", "wled", "spi refclk", "i2c";
CF-WR800N.dts-                  ralink,function = "gpio";
--
MLW221.dts:                     ralink,group = "i2c", "ephy", "wled";
MLW221.dts-                     ralink,function = "gpio";
--
MLWG2.dts:                      ralink,group = "i2c", "ephy", "wled";
MLWG2.dts-                      ralink,function = "gpio";
--
RT-N14U.dts:                    ralink,group = "ephy", "wled", "i2c";
RT-N14U.dts-                    ralink,function = "gpio";
--
WRH-300CR.dts:                  ralink,group = "i2c", "ephy", "wled";
WRH-300CR.dts-                  ralink,function = "gpio";
--
WT3020.dtsi:                    ralink,group = "ephy", "wled", "pa", "i2c", "wdt", "uartf";
WT3020.dtsi-                    ralink,function = "gpio";
--
ZBT-WA05.dts:                   ralink,group = "i2c", "spi refclk", "wled";
ZBT-WA05.dts-                   ralink,function = "gpio";
--
ZBT-WE2026.dts:                 ralink,group = "i2c", "spi refclk", "wled";
ZBT-WE2026.dts-                 ralink,function = "gpio";
--
ZBT-WR8305RT.dts:                       ralink,group = "i2c", "uartf", "spi refclk", "wled";
ZBT-WR8305RT.dts-                       ralink,function = "gpio";
--
kn_rc.dts:                      ralink,group = "i2c", "wdt", "pa", "spi refclk", "wled";
kn_rc.dts-                      ralink,function = "gpio";
--
kn_rf.dts:                      ralink,group = "i2c", "wdt", "pa", "spi refclk", "wled";
kn_rf.dts-                      ralink,function = "gpio";

Therefore all the file using the <&gpio3 0> is because explicitly use that pin as a GPIO overriding the standard hw controlled wled function.